include/glow/Optimizer/Optimizer.h

```diff
@@ -51,7 +52,13 @@ void lower(Function *F, CompilationContext &cctx, const Backend *B = nullptr,
            const KindSet &doNotLowerKinds = {});
 
 /// Dead code elimination.
-void DCE(Function *F);
+class DCE : public FunctionPass {
+private:
+  bool passImpl(Function *F);
```

Comment:
I haven't seen where the return value is used.  Will it used in the future? If so, maybe some comments could be added.

Reply:
This is used by the pass to signify if it changed the Function. Some of our passes do this, so I've set it up so all will.